vaapi/vaapicontext.cpp

```diff
@@ -24,8 +24,10 @@
 #include "common/common_def.h"
 #include "vaapi/vaapidisplay.h"
 #include "vaapi/VaapiUtils.h"
+#include "vaapi/vaapistreamable.h"
 #include <algorithm>
 #include <vector>
+#include <sstream>
```

Comment:
This is already included indirectly via vaapistreamable.h.  It is not even needed in this unit if you use toString(...) instead (which you just added in vaapistreamable.h).